### ISSUE TITLE
Wrong receipe for Rare Power Transformer XL

### ISSUE THREAD

**Issue by zer0fg** [yamamushi/dumap#128]
The recipe for rare power transformer XL is wrong, it now require 2401 maraging steel on top of all the magnet and components.
I believe your recipe says 343.
![RPTXL](https://user-images.githubusercontent.com/26676350/98141632-200d4d80-1ec7-11eb-8695-dd6743bead94.png)

**Comment by AdrienTODA** [comment]
Yep, ran into this issue today. The craft planner says 900L of Maraging Steel, but it needs 2401L.
